mesheryctl/pkg/utils/testing.go

```diff
@@ -189,10 +194,36 @@ func StopMockery(t *testing.T) {
 	httpmock.DeactivateAndReset()
 }
 
-// Set file location for testing stuff
+// Set file locations for few meshery files
 func SetFileLocationTesting(t *testing.T, dir string) {
 	MesheryFolder = filepath.Join(dir, "fixtures", MesheryFolder)
 	DockerComposeFile = filepath.Join(MesheryFolder, DockerComposeFile)
 	AuthConfigFile = filepath.Join(MesheryFolder, AuthConfigFile)
-	DefaultConfigPath = filepath.Join(MesheryFolder, DefaultConfigPath)
+	// Not setting DefaultConfigPath as that has been covered in SetupCustomContextEnv() and SetupContextEnv()
+}
+
+// repopulate config yaml
+func Copy(src, dst string) error {
+	sourceFileStat, err := os.Stat(src)
+	if err != nil {
```

Comment:
Can we rename it to something appropriate and have some description just above the command explaining what it is.

Reply:
Didn't add comment because I thought the function signature was pretty self-explanatory.
Any other suggestions for name

Reply:
what's the idea behind this function?

Reply:
copy all of the content from the file at src path to the file at dest path.

Reply:
What's the use-case, I mean why it was created?

Reply:
Basically the flow for any unit test that changes the test yamls is this-
1. We have a static yaml that doesn't change-call its Y1
2. We have a test yaml that changes during test- call it Y2(is a copy of Y1)
3. We run the command during test, it changes Y2. We check if the modified Y2 is what we expect by comparing it to a golden file.
4. Everything checks out, but now Y2 is modified. So before exiting,
5. We refill the Y2 with contents from Y1(with copy function), so that Y2 is at the same state as it was before running the tests.

Reply:
So it's like Reinstating a file back to the original. Wow, rename it. I know it's a tough one but let's see your creative mind here!

// @leecalcote @navendu-pottekkat any ideas??

Reply:
how about, "populateYaml"?